From a307c9727be4b4b608b5e6b1ae3f46218df479c2 Mon Sep 17 00:00:00 2001
From: Johannes Demel <demel@uni-bremen.de>
Date: Sat, 2 Oct 2021 11:38:46 +0200
Subject: [PATCH 56/73] format: Fix code format

I was too quick to merge a PR and missed a formatting issue. Thus, I fix
it now.

Signed-off-by: Johannes Demel <demel@uni-bremen.de>
---
 lib/volk_malloc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/volk_malloc.c b/lib/volk_malloc.c
index f489ef8..0410d29 100644
--- a/lib/volk_malloc.c
+++ b/lib/volk_malloc.c
@@ -56,8 +56,8 @@ void* volk_malloc(size_t size, size_t alignment)
     }
     // Tweak size to satisfy ASAN (the GCC address sanitizer).
     // Calling 'volk_malloc' might therefor result in the allocation of more memory than
-    // requested for correct alignment. Any allocation size change here will in general not
-    // impact the end result since initial size alignment is required either way.
+    // requested for correct alignment. Any allocation size change here will in general
+    // not impact the end result since initial size alignment is required either way.
     if (size % alignment) {
         size += alignment - (size % alignment);
     }
-- 
2.30.2

